messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
```

Comment:
- maybe "used to reserve (to lock) tokens for a mediated transfer"
- do we have a link to an explanation of the term "mediated transfer"?

Reply:
I am not sure, I will have to check.